docs/includes/apiargs-MongoDBCollection-method-watch-option.yaml

```diff
@@ -0,0 +1,36 @@
+source:
+  file: apiargs-MongoDBCollection-method-aggregate-option.yaml
+  ref: batchSize
+---
+source:
+  file: apiargs-MongoDBCollection-common-option.yaml
+  ref: collation
+---
+arg_name: option
+name: fullDocument
+type: string
+description: |
+  Allowed values are 'default' and 'updateLookup'. Defaults to 'default'.
+  When set to 'updateLookup', the change notification for partial updates will
+  include both a delta describing the changes to the document, as well as a
+  copy of the entire document that was changed from some time after the change
+  occurred.
```

Comment:
We can add also a reference to the constants here, as was done for [`findOneAndUpdate()`](https://github.com/mongodb/mongo-php-library/blob/master/docs/includes/apiargs-MongoDBCollection-method-findOneAndUpdate-option.yaml#L24).

Unlike `findOneAndUpdate()`, I think we can leave this paragraph as is (without adding lengthy references to constants within) and just add the list to the bottom:

```
...after the change occurred. The following values are supported:

  - ``MongoDB\Operation\ChangeStream::FULL_DOCUMENT_DEFAULT`` (*default*)
  - ``MongoDB\Operation\ChangeStream::FULL_DOCUMENT_UPDATE_LOOKUP``
```